src/main/kotlin/com/terraformation/backend/accelerator/model/VoteModel.kt

```diff
@@ -0,0 +1,56 @@
+package com.terraformation.backend.accelerator.model
+
+import com.terraformation.backend.db.accelerator.CohortPhase
+import com.terraformation.backend.db.accelerator.VoteOption
+import com.terraformation.backend.db.accelerator.tables.pojos.ProjectVotesRow
+import com.terraformation.backend.db.accelerator.tables.references.PROJECT_VOTES
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.UserId
+import org.jooq.Record
+
+data class VoteModel(
+    val projectId: ProjectId,
+    val phase: CohortPhase,
+    val userId: UserId,
+    val voteOption: VoteOption?,
+    val conditionalInfo: String?
+) {
+  companion object {
+    fun of(
+        record: Record,
+    ): VoteModel {
+      return VoteModel(
+          projectId = record[PROJECT_VOTES.PROJECT_ID]!!,
+          phase = record[PROJECT_VOTES.PHASE_ID]!!,
+          userId = record[PROJECT_VOTES.USER_ID]!!,
+          voteOption = record[PROJECT_VOTES.VOTE_OPTION_ID],
+          conditionalInfo = record[PROJECT_VOTES.CONDITIONAL_INFO],
+      )
+    }
+
+    fun create(
+        projectId: ProjectId,
+        phase: CohortPhase,
+        userId: UserId,
+        voteOption: VoteOption?,
+        conditionalInfo: String?
```

Comment:
What's the advantage of calling this vs. directly using the constructor?

Reply:
Removed.